src/api.ts

```diff
@@ -136,9 +136,9 @@ export const connect = async (essid: string, password?: string): Promise<Connect
   );
 
 // call the API to start the hotspot mode
-export const startAp = async (): Promise<any> =>
+export const startHotspot = async (): Promise<any> =>
   callApi(
-    "/start-ap",
+    "/start-hotspot",
```

Comment:
You can't just change the API endpoint on the frontend side and leave it as it in the backend. It will break terribly.

Either you also update the backend (which can be complicated if other things use this endpoint)
Either you just keep it as it.

My request was ONLY to rename it "hotspot" for the user. The technical terms can stay as they were (access point/ap)